astroquery/linelists/cdms/core.py

```diff
@@ -303,12 +331,66 @@ def tryfloat(x):
 CDMS = CDMSClass()
 
 
+def parse_letternumber(st):
```

Comment:
Maybe add `_` as this is not suppose to end up in the public API (and indeed it's not added to __all__)